rdr_service/resource/schemas/genomics.py

```diff
@@ -93,12 +93,11 @@ class GenomicSetMemberSchema(Schema):
     cvl_w4f_manifest_job_run_id = fields.Int32()
     genomic_workflow_state = fields.EnumString(enum=GenomicWorkflowState)
     genomic_workflow_state_id = fields.EnumInteger(enum=GenomicWorkflowState)
-
-    genomic_workflow_state_history = fields.JSON()
+    # genomic_workflow_state_history = fields.JSON()
```

Comment:
What triggered the commenting out of the fields in this table?

Reply:
I commented out those fields when I found they had been removed from the RDR table.  I have removed them in my next PR.